index.js

```diff
@@ -292,7 +299,7 @@ function unregister(glob) {
     }
   }
   if (glob.Date === MockDate) {
-    assert(_Date);
+    assert.ok(_Date, 'need to pass a date');
```

Comment:
More accurate message is probably "unregister() called but missing original Date constructor".